pom.xml

```diff
@@ -5,19 +5,25 @@
     <parent>
         <groupId>life.qbic</groupId>
         <artifactId>parent-pom</artifactId>
-        <version>2.4.0-SNAPSHOT</version>
+        <version>3.0.0</version>
```

Comment:
do we want to keep the parent pom? I thought we remove it for osgi projects 🤔

Reply:
Good Catch! However I'd rather keep it in the current state for this PR and remove the Parent Pom in a seperate PR since this PR is pretty big already.

Reply:
I think it is really important therefore I want to make sure this really will be tackled at some point